packages/components/psammead-embed-error/src/index.jsx

```diff
@@ -0,0 +1,69 @@
+import React from 'react';
+import { string, bool } from 'prop-types';
+import styled from 'styled-components';
+import { C_CHALK, C_EBON } from '@bbc/psammead-styles/colours';
+import { GEL_SPACING_TRPL, GEL_SPACING } from '@bbc/gel-foundations/spacings';
+import { getSansRegular } from '@bbc/psammead-styles/font-styles';
+import { GEL_BODY_COPY } from '@bbc/gel-foundations/typography';
+import { BBC_BLOCKS, coreIcons } from '@bbc/psammead-assets/svgs';
+
+const GEL_SPACING_QRT = '0.125rem';
+
+const StyledEmbedError = styled.div`
+  ${({ service }) => getSansRegular(service)}
+  ${GEL_BODY_COPY};
+  background-color: ${C_CHALK};
+  background-image: url(data:image/svg+xml;base64,${BBC_BLOCKS});
+  background-position: center center;
+  background-repeat: no-repeat;
+  background-size: 38.2%;
```

Comment:
Might be worth commenting the magic number to avoid head-scratching.

Reply:
Unfortunately stylelint throws an error if it finds a comment in a Styled Component block, so I've moved the `38.2%` to a named constant, which is probably better anyway.